src/components/start.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+
+import CustomNavbar from './custom-nav-bar.js';
+import CustomBottomNavbar from './custom-bottom-nav-bar.js';
+
+import '../assets/styles/start.css'
+import start_img_1 from  '../assets/images/start1.png'
+import company_chat from '../assets/images/company-chat.png'
+import lunch_together from '../assets/images/lunch-together.png'
+import one_on_one from '../assets/images/one-on-one.png'
+
+
+class Start extends React.Component {
+    render() {
+        return (
+            <>
+                <CustomNavbar />
+                <header className={"header-wrapper"} id={"about"}>
+                    <div className={"header-solid"}>
+                        <div className={"header-content content"}>
+                            <div className={"header-copy"}>
+                                <h1 className={"header-h1 smaller-text-space"}>Gauss Talk</h1>
+                                <p className>
+                                    A platform where Gaussians can communicate, discuss, and share their different ideas and improve company culture
+                                </p>
+                                <div className={"btn-group text-center"}>
+                                    <Button href="https://login.microsoftonline.com/cfcd9b87-7c5a-4042-9129-abee6253febe/oauth2/v2.0/authorize?client_id=7fc37514-c400-4b28-a6d6-e19a9ae981b6&response_type=code&redirect_uri=http://localhost:3000/auth&scope=User.read" target="_self" variant={"contained"}>
```

Comment:
redirect_uri is localhost oh no. The link is on the other file too. I had to find it at the first time.

Reply:
Should be https://talk.gausslabs.ai/home/auth right now.